command.ts

```diff
@@ -0,0 +1,232 @@
+    },
+    changelogHeader: {
+      string: true,
+      describe: 'This option will be removed in the next major version',
+      deprecated: 'Use --header'
+    },
+    preset: {
+      string: true,
+      // default: defaults.preset,
+      describe: 'Commit message guideline preset'
+    },
+    lernaPackage: {
+      alias: 'lerna-package',
+      string: true,
+      describe: 'Name of the package from which the tags will be extracted'
+    },
+    npmPublishHint: {
+      string: true,
+      // default: defaults.npmPublishHint,
+      describe: 'Customized publishing hint'
+    },
+    header: {
+      alias: 'h',
+      string: true,
+      // default: defaults.header,
+      describe: 'Use a custom header when generating and updating changelog',
+    },
+    types: {
+      alias: 'type',
+      array: true,
+      // default: defaults.types,
+      describe: spec.types.description
+    },
+    preMajor: {
+      boolean: true,
+      // default: defaults.dryRun,
+      describe: spec.preMajor.description,
+    },
+    commitUrlFormat: {
+      string: true,
+      describe: spec.commitUrlFormat.description,
+    },
+    issueUrlFormat: {
+      string: true,
+      describe: spec.issueUrlFormat.description,
+    },
+    userUrlFormat: {
+      string: true,
+      describe: spec.userUrlFormat.description,
+    },
+    issuePrefixes: {
+      array: true,
+      string: true,
+      describe: spec.issuePrefixes.description,
+    },
+  })
+  // .check((argv) => {
+  //   if (argv.scripts && (typeof argv.scripts !== 'object' || Array.isArray(argv.scripts))) {
+  //     throw Error('scripts must be an object')
+  //   } else if (argv.skip && (typeof argv.skip !== 'object' || Array.isArray(argv.skip))) {
+  //     throw Error('skip must be an object')
+  //   } else {
+  //     return true
+  //   }
+  // })
+  .example('$0', 'Update changelog and tag release')
+  .example(
+    '$0 -m "%s: see changelog for details"',
+    'Update changelog and tag release with custom commit message'
+  )
+  // .pkgConf('standard-version')
+  // .pkgConf('commit-and-tag-version')
+  // .config(getConfiguration())
+  .wrap(97);
+
+// type YargsOptionsType = 'array' | 'count' | 'boolean' | 'number' | 'string' | undefined;
+
+// function convertSpecPropertyType(type: JSONSchema7TypeName): YargsOptionsType {
+//   if (type === 'integer') return 'number';
+//   if (type === 'null') return undefined;
+//   if (type === 'object') return 'array';
+//   return type;
+// }
+
+// Object.keys(spec).forEach((propertyKey) => {
+//   const propName = propertyKey as keyof typeof spec;
+//   const property = spec[propName];
+//   console.log('propertyKey:', propertyKey);
+
+//   cmdParser.option(propertyKey, {
+//     type: convertSpecPropertyType(property.type),
+//     describe: property.description,
+//     // default: defaults[propName] ? defaults[propName] : property.default,
+//     // group: 'Preset Configuration:',
+//   });
+// });
```

Comment:
There are a lot of comments in this file - are they supposed to be still commented?

Reply:
It was the end of my day and I wanted to get the changes pushed up to get other eyes on them so I was in a hurry and didn't do any cleanup.  :)